example/example18.cpp

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Please make example18 a normal part of the extension library, just like example1-17.